src/heartratetask/HeartRateTask.cpp

```diff
@@ -5,8 +5,25 @@
 
 using namespace Pinetime::Applications;
 
-HeartRateTask::HeartRateTask(Drivers::Hrs3300& heartRateSensor, Controllers::HeartRateController& controller)
-  : heartRateSensor {heartRateSensor}, controller {controller} {
+TickType_t CurrentTaskDelay(HeartRateTask::States state, TickType_t ppgDeltaTms) {
+  switch (state) {
+    case HeartRateTask::States::ScreenOnAndMeasuring:
+    case HeartRateTask::States::ScreenOffAndMeasuring:
+      return ppgDeltaTms;
+    case HeartRateTask::States::ScreenOnAndStopped:
+      return pdMS_TO_TICKS(100);
```

Comment:
Can this be `portMAX_DELAY`? If the screen is on and the measuring is stopped, is it true that the only event that causes a change is `StartMeasurement`?

Reply:
Changed to `portMAX_DELAY`